deploy/olm-catalog/windows-machine-config-operator/manifests/prometheusRule.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: win-prometheus-k8s-rules
+spec:
+  groups:
+    - name: windows.rules
+      rules:
+        - expr: |
+            avg without (core, mode)(rate(windows_cpu_time_total{mode="idle"}[1m]))
+          record: instance:node_cpu_utilisation:rate1m
+        - expr: |
+            sum(rate(windows_cpu_time_total{mode="idle",mode!="iowait"}[3m])) BY (instance)
+          record: instance:node_cpu:rate:sum
+        - expr: |
+            windows_logical_disk_size_bytes
```

Comment:
I assume `windows_logical_disk_size_bytes` is something `windows_exporter` is exposing.  

A logical disk needn't have all its space allocated. Are there other options for disk size that `windows_exporter` exposes?

Reply:
Did not see any other options for disk-size that can be used in `windows-exporter`
```
#HELP windows_logical_disk_free_bytes Free space in bytes (LogicalDisk.PercentFreeSpace)
# HELP windows_logical_disk_idle_seconds_total Seconds that the disk was idle (LogicalDisk.PercentIdleTime)
# HELP windows_logical_disk_read_bytes_total The number of bytes transferred from the disk during read operations (LogicalDisk.DiskReadBytesPerSec)
# HELP windows_logical_disk_read_latency_seconds_total Shows the average time, in seconds, of a read operation from the disk (LogicalDisk.AvgDiskSecPerRead)
# HELP windows_logical_disk_read_seconds_total Seconds that the disk was busy servicing read requests (LogicalDisk.PercentDiskReadTime)
# HELP windows_logical_disk_read_write_latency_seconds_total Shows the time, in seconds, of the average disk transfer (LogicalDisk.AvgDiskSecPerTransfer)
# HELP windows_logical_disk_reads_total The number of read operations on the disk (LogicalDisk.DiskReadsPerSec)
# HELP windows_logical_disk_requests_queued The number of requests queued to the disk (LogicalDisk.CurrentDiskQueueLength)
# HELP windows_logical_disk_size_bytes Total space in bytes (LogicalDisk.PercentFreeSpace_Base)
# HELP windows_logical_disk_split_ios_total The number of I/Os to the disk were split into multiple I/Os (LogicalDisk.SplitIOPerSec)
# HELP windows_logical_disk_write_bytes_total The number of bytes transferred to the disk during write operations (LogicalDisk.DiskWriteBytesPerSec)
# HELP windows_logical_disk_write_latency_seconds_total Shows the average time, in seconds, of a write operation to the disk (LogicalDisk.AvgDiskSecPerWrite)
# HELP windows_logical_disk_write_seconds_total Seconds that the disk was busy servicing write requests (LogicalDisk.PercentDiskWriteTime)
# HELP windows_logical_disk_writes_total The number of write operations on the disk (LogicalDisk.DiskWritesPerSec)
```